### DISCUSSION TITLE
Welcome to windowbuilder Discussions!

### DISCUSSION THREAD

**Discussion by lcaron** [eclipse/windowbuilder#124] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Feel free to introduce yourself and start a discussion on any of the available topics.

 The WindowBuilder Team

**Comment by KokyMk** [comment]
![error](https://github.com/eclipse-windowbuilder/windowbuilder/assets/162581832/d888a560-6a4d-4465-9963-64ce202ffd5b)
Can u plz help me with this problem

**Reply by ptziegler** [reply]
This post here isn't really the right place to ask such questions, so I kindly ask that we move any further discussion to #656.

Regardless, you seem to be having two conflicting WindowBuilder installations in your workspace, the 1.9.7 and a more recent version. In order to fix this problem, the older version needs to be removed.
In the installation history, you would have to revert to a state _before_ the 1.9.7 was installed and then simply install the latest version instead. Alternatively, you can also just set up a clean Eclipse IDE and install WindowBuilder there.
